wayland.c

```diff
@@ -100,16 +100,18 @@ static void touch_handle_motion(void *data, struct wl_touch *wl_touch,
 		uint32_t time, int32_t id,
 		wl_fixed_t surface_x, wl_fixed_t surface_y) {
 	struct mako_seat *seat = data;
-	seat->touch.x = wl_fixed_to_int(surface_x);
-	seat->touch.y = wl_fixed_to_int(surface_y);
+	if (id >= MAX_TOUCHPOINTS) return;
```

Comment:
Style nit: This should be broken out to a block with braces in all three spots.

Reply:
Done!